docs/projects/fps/tutorial.md

```diff
@@ -84,19 +84,21 @@ public static EntityTemplate HealthPickup(Vector3f position, uint healthValue)
 {
     var gameLogic = WorkerUtils.UnityGameLogic;
 
-    var healthPickupComponent = Pickups.HealthPickup.Component.CreateSchemaComponentData(true, healthValue);
-
-    return EntityBuilder.Begin()
-        .AddPosition(position.X, position.Y, position.Z, gameLogic)
-        .AddMetadata("HealthPickup", gameLogic)
-        .SetPersistence(true)
-        .SetReadAcl(AllWorkerAttributes)
-        .AddComponent(healthPickupComponent, gameLogic)
-        .Build();
+    var healthPickupComponent = new Pickups.HealthPickup.Snapshot { IsActive = true, HealthValue = healthValue };
+
+    var entityTemplate = new EntityTemplate();
+    entityTemplate.AddComponent(new Position.Snapshot { Coords = new Coordinates(position.X, position.Y, position.Z) }, gameLogic);
+    entityTemplate.AddComponent(new Metadata.Snapshot { EntityType = "HealthPickup"}, gameLogic);
+    entityTemplate.AddComponent(new Persistence.Snapshot(), gameLogic);
+    entityTemplate.AddComponent(healthPickupComponent, gameLogic);
+    entityTemplate.SetReadAccess(gameLogic, WorkerUtils.UnityClient);
+    entityTemplate.SetComponentWriteAccess(EntityAcl.ComponentId, gameLogic);
+
+    return entityTemplate;
 }
 ```
 
-The `EntityBuilder` syntax provides a compact way to declare the relevant components. You may notice that `.AddPosition`, `.AddMetadata`, and `.SetPersistence` appear in the entity template function of _every_ entity type. This is because these are mandatory "well-known components" that SpatialOS expects.
+You may notice that `Position` and `Metadata` appear in the entity template of _every_ entity type. This is because these are mandatory "well-known components" that SpatialOS expects. `Persistence` is another "well-known component" that is optional.
```

Comment:
```suggestion
You may notice that `Position` and `Metadata` appear in the entity template of _every_ entity type. This is because these are standard-library components that SpatialOS expects. (Note that `Persistence` is another standard-library component but this is optional.)
```
Suggest removing mandatory as it is easily confused with `required components`. You've used "SpatialOS expects" so it's clear we need it.

Reply:
Done!